src/scripts/main.js

```diff
@@ -1,3 +1,10 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+spider.style.top =
+  wall.clientHeight - wall.clientHeight / 2 - spider.offsetHeight / 2 + 'px';
+
+spider.style.left =
+  wall.clientWidth - wall.clientWidth / 2 - spider.offsetWidth / 2 + 'px';
```

Comment:
The calculation for 'spider.style.left' seems incorrect. It should be 'wall.clientWidth / 2 - spider.offsetWidth / 2' to center the spider horizontally within the wall.